components/layout_2020/display_list/mod.rs

```diff
@@ -404,6 +422,10 @@ impl<'a> BuilderForBoxFragment<'a> {
         self.build_hit_test(builder);
         self.build_background(builder);
         self.build_border(builder);
+        // FIXME: the spec allows outlines to be either in-band or out-of-band.
+        // Drawing them here is very similar to in-band, but not exactly,
+        // and out-of-band would be better.
+        self.build_outline(builder);
```

Comment:
Oh, sorry, one more step:
- Sort the new section as step 10 at the end of in `stacking_context::StackingContext::build_display_list()`.

Reply:
Thanks, I will attempt that

Reply:
Do the background and border need to be handled for `StackingContextSection::Content`? What about hit-test?

Reply:
I don't think so, because currently it's only used for text, images, and iframes (which just add a display list item that is a portal into another display list).

Reply:
Lots of tests failed when I tried. I thought it was just for raw text contents which don't have a style of their own, but this is used for inline boxes, they can have some border or background.